highfidelity/hifi
# [Home] Add More Content

This PR adds more content to the "Home" space:

- Fire pit that you can toss objects into
- Various moustaches and beards in the dressing room
- A bench in the hallway
- An umbrella outside
- Removes blue chair
- Swaps switches in living room
- Removes generic block pile from table
- Adds blocky game and 'advance' swipe button
- New artemis model with jacket
- Adds milk pail game and balls to reset script
- Adds target with auto reset after collision and tipped over to ping pong gun

To test:
0. Stop all scripts and load the defaultScripts.js file that comes with this install
1. Visit forum
2. Reload all scripts (Running scripts / Ctrl-R if advanced settings is enabled)
3. Click the tidy guy
4. Observe that objects above reset as expected
